package.json

```diff
@@ -67,10 +67,11 @@
     "cozy-client": "35.3.1",
     "cozy-doctypes": "^1.82.2",
     "cozy-flags": "^2.7.2",
+    "cozy-intent": "^2.14.0",
```

Comment:
Why is it necessary?

Reply:
Otherwise I get this error:

```
Uncaught Error: Cannot find module 'cozy-intent'
    at webpackMissingModule (useSetFlagshipUI.js:1:1)
    at eval (useSetFlagshipUI.js:1:1)
    at ./node_modules/cozy-ui/transpiled/react/hooks/useSetFlagshipUi/useSetFlagshipUI.js (dissecozy.js:8456:1)
    at __webpack_require__ (dissecozy.js:785:30)
    at fn (dissecozy.js:151:20)
    at eval (index.js:1:1)
    at ./node_modules/cozy-ui/transpiled/react/Sidebar/index.js (dissecozy.js:8377:1)
    at __webpack_require__ (dissecozy.js:785:30)
    at fn (dissecozy.js:151:20)
    at Module.eval (Sidebar.jsx:8:90)
```

Reply:
I assume this is a BC from cozy-ui related to the Sidebar